src/spinnaker_ros_interface_main.cc

```diff
@@ -310,6 +330,20 @@ void CaptureLoop(CameraPtr pCam) {
   }
 }
 
+bool SetCameraInfoSrvCallback(sensor_msgs::SetCameraInfo::Request &request, sensor_msgs::SetCameraInfo::Response &response)
```

Comment:
The service is not full integrated with the ability to actually change the camera parameters during run time, it only loads the calibration matrices which is the data required by the utilities.